platform/lib/pipeline/componentReferenceImporter.js

```diff
@@ -72,9 +74,13 @@ class ComponentReferenceImporter {
     // Keep track of all promises to complete function
     const savedDocuments = [];
 
+    const versionMapping = {};
     for (const extension of extensions) {
       const documents = await this._findExtensionDocs(extension);
       const versions = [...new Set(documents.map((doc) => doc.version).sort().reverse())];
+      const supportedVersions = {};
+      versionMapping[extension.name] = supportedVersions;
```

Comment:
We should move this into the else below to only add the components where we have documentation for. Otherwise there are quite a few useless, empty components in the json.